src/Selenium2Driver.php

```diff
@@ -189,14 +189,7 @@ public static function getDefaultCapabilities()
     {
         return array(
             'browserName'       => 'firefox',
-            'version'           => '9',
-            'platform'          => 'ANY',
```

Comment:
Following similar reasoning as my previous comment, can this capability also be left in? This seems a reasonable default, and it would be convenient and intuitive to allow users to leave it out of their own capabilities when they don't care. Thoughts?

Reply:
@stof Any response to this? `'ANY'` is likely needed if we don't want to force the user to explicitly specify the platform when it doesn't matter.